### ISSUE TITLE
Map colour code/key

### ISSUE THREAD

**Issue by scottofletcher** [participedia/frontend#797]
From Collaborators at KWI:

"The key under the map is missing, which colour stand[s] for what entry?"

**Comment by jesicarson** [comment]
Hmm.. good to know it's not clear that red pins represent featured or filtered content. Need to think about how to clarify this. Legend is possible, but would people understand "red pins = featured or filtered content" ? Is there a more clear way to say this? I'm pretty sure its in the feature tour... and that reminds me, there's no way to reload the feature tour on xyz right now.

**Comment by scottofletcher** [comment]
This is complicated but could we make the map more interactive? My dream has always been to be able to select from a menu stuff like 'featured cases', 'all cases', 'participatory budgeting', 'ongoing events', etc. I know the Ash Center's Data Smart section features lots of mapping projects - perhaps some of them are open source? Here's a link: https://datasmart.ash.harvard.edu/civic-data/map-month

**Comment by jesicarson** [comment]
Definitely, the idea for the map area is that once the main parts of the site are running we can use that as an experimental space for data visualizations, etc. Would be nice to get to an experimental space soon.

**Comment by scottofletcher** [comment]
Love it. I'm weirdly obsessed with data visualisations and info graphics! Especially when they're interactive :P
